client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -808,6 +811,30 @@
                     });
                 };
 
+                /**
+                 * Export Highlight related
+                 */
+                $scope.exportHighlight = function(item) {
+                    if ($scope.save_enabled()) {
+                        modal.confirm(gettext('You have unsaved changes, do you want to continue.'))
+                            .then(function() {
+                                _exportHighlight(item._id);
+                            }
+                        );
+                    } else {
+                        _exportHighlight(item._id);
+                    }
+                };
+
+                function _exportHighlight(_id) {
```

Comment:
Docstring is missing... we write them for private functions, too, AFAIK (but please correct me if I'm wrong).